src/LanguageServer.php

```diff
@@ -289,10 +324,48 @@ public function initialize(ClientCapabilities $capabilities, string $rootPath =
         });
     }
 
+    /**
+     * The initialized notification is sent from the client to the server after
+     * the client received the result of the initialize request but before the
+     * client is sending any other request or notification to the server.
+     *
+     * @return Promise
+     */
+    public function initialized(): Promise
+    {
+        return coroutine(function () {
+            if (!$this->rootPath) {
+                return;
+            }
+
+            // request configuration if it is supported
+            // support comes with protocol version 3.6.0
+            if ($this->clientCapabilities->workspace->configuration) {
+                $configurationitem = new ConfigurationItem();
+                $configurationitem->section = 'php';
+                $configuration = yield $this->client->workspace->configuration([$configurationitem]);
+                $options = $this->mapper->map($configuration[0], new Options());
+            }
+
+            // depending on the implementation of the client
+            // the workspace/didChangeConfiguration can be invoked before
+            // the response from the workspace/configuration request is resolved
+            if ($this->indexer->isIndexing()) {
+                return;
+            }
+
+            if ($options) {
+                $this->indexer->setOptions($options);
+            }
+
+            $this->indexer->index()->otherwise('\\LanguageServer\\crash');
```

Comment:
You should `yield` this instead and make the whole coroutine crash on error since notifications can't return error responses